server/service/src/requisition/request_requisition/insert.rs

```diff
@@ -133,10 +133,12 @@ fn generate(
         // Default
         sent_datetime: None,
         approval_status: None,
-        program_id: None,
         finalised_datetime: None,
         linked_requisition_id: None,
         is_sync_update: false,
+        program_id: None, // TODO: Add program_id to input
```

Comment:
i think it would be another end point, since logic is a bit different and some things come from order type (could keep it as one endpoint though, but I think it would be easier to reason about if it's separate, even though there would be some duplication)